Could not find a declaration file for module 'react-daisyui'

Typings works perfectly if for example I include "react-daisy-ui" in a NextJS app.
But they are not working [in a project where I am using ESM](https://github.com/giacomorebonato/Jack-Fastify) (I believe this is somehow the cause of the issue).

```
[1] TSError: ⨯ Unable to compile TypeScript:
[1] src/components/navigation.tsx(1,24): error TS7016: Could not find a declaration file for module 'react-daisyui'. '/users/<home>/mah-fastify/node_modules/react-daisyui/dist/react-daisyui.modern.js' implicitly has an 'any' type.
```

And of course I see that typings are present, so I don't really understand where the problem is coming from.
Do you have any idea?

--- a/package.json
+++ b/package.json
@@ -12,7 +12,8 @@
   "exports": {
     "import": "./dist/react-daisyui.modern.js",
     "require": "./dist/react-daisyui.cjs",
-    "default": "./dist/react-daisyui.modern.js"
+    "default": "./dist/react-daisyui.modern.js",
+    "types": "./dist/index.d.ts"
   },
   "main": "./dist/react-daisyui.cjs",
   "module": "./dist/react-daisyui.esm.js",

